### PR TITLE
Vet clinic database: Query and update animals table

### PR DESCRIPTION
**Pull request by ernestmusong**
## The following have been done to the animals table:

- Add a column **species** of type **string** to the animals table.
- Modify schema.sql file.
- Insert data into data.sql.
- Add transactions in queries.sql.
- Write queries to answer questions.

**All screenshots corresponding to the above queries are found [here](https://github.com/ernestmusong/Vet-clinic-database/tree/alter-table/screenshots/query-and-update%20animals-table/Queries)**

**All screenshots corresponding to the above transactions are found [here](https://github.com/ernestmusong/Vet-clinic-database/tree/alter-table/screenshots/query-and-update%20animals-table/Transactions)**

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,118 @@
...
+FROM animals
+GROUP BY species
+HAVING species = 'pokemon';
+
+SELECT
+species,
+MAX(weight_kg)
+FROM animals
+GROUP BY species
+HAVING species = 'pokemon';
```
Comment (brytebee) [inline queries.sql]: Kindly write a single query to solve this challenge.

Consider the code snippet below.

```
SELECT species, MIN(weight_kg), MAX(weight_kg)
FROM animals
GROUP BY species;
```

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ernestmusong 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB setup.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (ernestmusong) [reply queries.sql]: > Kindly write a single query to solve this challenge.
> 
> Consider the code snippet below.
> 
> ```
> SELECT species, MIN(weight_kg), MAX(weight_kg)
> FROM animals
> GROUP BY species;
> ```

Thanks very much @brytebee for this feedback, [here is the correct result](https://github.com/ernestmusong/Vet-clinic-database/blob/alter-table/screenshots/query-and-update%20animals-table/Queries/What%20is%20the%20minimum%20and%20maximum%20weight%20of%20each%20type%20of%20animal.png)

**Review by BertrandConxy — APPROVED** [review]
Hi @ernestmusong, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**